requirements.txt

```diff
@@ -2,3 +2,9 @@ xlrd==1.0.0
 python-magic==0.4.12
 messytables==0.15.2
 progressbar==2.3
+SQLAlchemy>=0.6.6
+requests==2.11.1
+six==1.11.0
+
+-e git+https://github.com/ckan/ckanext-archiver.git@2.0.0#egg=ckanext-archiver
```

Comment:
how should we handle this as it might clobber the true version we want.

Reply:
Good catch, our fork of ckanext-qa should use a fork of ckanext-archiver here.